src/Servers/Kestrel/Transport.Quic/src/QuicTransportOptions.cs

```diff
@@ -27,21 +27,6 @@ public class QuicTransportOptions
         /// </summary>
         public string? Alpn { get; set; }
 
-        /// <summary>
-        /// The certificate that MsQuic will use.
-        /// </summary>
-        public X509Certificate2? Certificate { get; set; }
-
-        /// <summary>
-        /// Optional path to certificate file to configure the security configuration.
-        /// </summary>
-        public string? CertificateFilePath { get; set; }
```

Comment:
@jkotalik What did these cert file path options do? I saw they were passed all the way into msquic

Reply:
These were used for limitations in msquic, not accepting a x509 cert handle at the time. We should remove this though; it was a temporary work around.

Reply:
We should confirm that either this now does work with msquic, or have a feature ask for them.